aws/rds_enhanced_monitoring/lambda_function.py

```diff
@@ -287,8 +289,15 @@ def lambda_handler(event, context):
 
 
 class Stats(object):
-    def __init__(self):
+    def __init__(self, base=2, cap=30, max_attempts=5):
         self.series = []
+        self.base = base
+        self.cap = cap
+        self.max_attempts = max_attempts
+
+    def _backoff(self, n):
+        v = min(self.cap, pow(2, n) * self.base)
+        return random.uniform(0, v)
```

Comment:
Ref:
- https://aws.amazon.com/blogs/architecture/exponential-backoff-and-jitter/
- https://github.com/aws-samples/aws-arch-backoff-simulator/blob/66cb169277051eea207dbef8c7f71767fe6af144/src/backoff_simulator.py#L40-L43